mozilla_aws_cli/role_picker.py

```diff
@@ -35,10 +41,12 @@ def output_set_env_vars(var_map, message=None):
         name = tempfile.mkstemp(suffix='.sh', prefix='maws-')[1]
         with open(name, 'w') as f:
             vars_to_set = [
-                "=".join((x, var_map[x]))
+                "=".join((x, str(var_map[x])))
                 for x in var_map if var_map[x] is not None]
             if vars_to_set:
                 f.write("export {}\n".format(" ".join(vars_to_set)))
+                f.write('alias maws-logout="unset {}"\n'.format(
+                    " ".join([x for x in var_map if var_map[x] is not None])))
```

Comment:
Mixed quotation marks.  :P

Reply:
This was because I had `"` inside the string